testsuite/features/support/commonlib.rb

```diff
@@ -209,7 +209,11 @@ def extract_logs_from_node(node)
   os_family = node.os_family
   node.run('zypper --non-interactive install tar') if os_family =~ /^opensuse/ && !$is_container_provider
   node.run('journalctl > /var/log/messages', check_errors: false)
-  node.run('venv-salt-call --local grains.items | tee -a /var/log/salt_grains', verbose: true, check_errors: false)
+  if node.full_hostname.include?('-srv')
+    node.run('salt-call --local grains.items | tee -a /var/log/salt_grains', verbose: true, check_errors: false)
+  else
+    node.run('venv-salt-call --local grains.items | tee -a /var/log/salt_grains', verbose: true, check_errors: false)
+  end
```

Comment:
A server in theory shouldn't be a salt minion itself.
So better if we do something like:
```suggestion
  unless $host_by_node[node] == 'server'
    node.run('venv-salt-call --local grains.items | tee -a /var/log/salt_grains', verbose: true, check_errors: false)
  end
```

Reply:
Thank you for the suggestion!